### DISCUSSION TITLE
LED1545G12 power_on_behavior toggle

### DISCUSSION THREAD

**Discussion by jagjordi** [Koenkk/zigbee2mqtt#14782] [Devices]
Hello,
I have seen that some devices support the option to toggle on power on by setting power_on_behavior=toggle.
However it seems that LED1545G12 only supports on, off, previous. Is there any way to add the toggle option?

**Comment by Koenkk** [comment]
It should be available in the latest dev branch.  (https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html)

**Reply by jagjordi** [reply]
That is so good! I can now use a normal spring loaded wall switch with the ikea lights.

I have noticed that sometimes if the press is very fast it doesn't "register" as a power cycle. Is there a way to decrease the threshold of time that is needed to be registered as a power cycle?

**Reply by Koenkk** [reply]
AFAIK not
